lib/tasks/router_data.rake

```diff
@@ -11,9 +11,9 @@ namespace :router_data do
 
     puts "Writing file with #{artefacts.count} redirects"
 
-    csv = "Source,Destination,Type\n"
+    csv = "Source,Destination,Type,Segments\n"
     csv << artefacts
-      .map { |arr| "/#{arr[0]},#{arr[1].gsub(/#.*/, '')},prefix" }
+      .map { |arr| "/#{arr[0]},#{arr[1]},prefix" }
```

Comment:
Should be ok as is. Keeping the same structure as [here](https://github.gds/gds/router-data/blob/5c446e3e2a31499cc0afdb19a8dcc91576c78dab/data/worldwide_priority_redirects.csv)